Dockerfile

```
@@ -1,34 +1,7 @@
-# Build the manager binary
-FROM golang:1.19 as builder
-ARG TARGETOS
-ARG TARGETARCH
+FROM gcr.io/distroless/static:debug-nonroot
```

I don't think we need the debug variant for operator-controller do we?

```suggestion
FROM gcr.io/distroless/static:nonroot
```